### DISCUSSION TITLE
Why are some plugins painted black and others grey?

### DISCUSSION THREAD

**Discussion by rediffusion** [xiaoxiaoflood/firefox-scripts#140] [Legacy Development]
Actually, I don't care but I want to know the reason!
Look:

![extensionOptionsMenu uc](https://user-images.githubusercontent.com/30071027/169032609-4d9f4308-ef9b-412a-9235-c26a01959953.png)

Respectfully Yours! (¬_¬")

**Comment by xiaoxiaoflood** [comment]
The grey ones don't have a proper options page/window, so it means left click is disabled, because left clicking an extension from this menu opens its settings.
